specification/devopsinfrastructure/Microsoft.DevOpsInfrastructure/main.tsp

```diff
@@ -353,6 +353,7 @@ model ResourceDetailsObject is ProxyResource<ResourceDetailsObjectProperties> {
   @segment("resources")
   @visibility("read")
   @path
+  @pattern("^[a-zA-Z0-9-]{3,24}$")
```

Comment:
I think we should probably suppress this rule unless this was checked with the service team